scripts/ci-e2e-gc.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
This is should be combined with the previous one and create a single script with a cli argument saying something like `{managed,unmanaged}`. WDYT?

Reply:
These scripts are only temporary while the feature is experimental. Ultimately, they will be removed and these tests will be incorporated into the managed and unmanaged suites.

So i basically haven't spent anytime optimizing this.

Reply:
ah, ok. makes sense then. Thanks.